website/react-magma-docs/src/pages/api/input.mdx

```diff
@@ -170,6 +170,8 @@ export function Example() {
 If an input has an `errorMessage`, the input will be styled to highlight it's error state and the error message will appear below the input field.
 If an error message is present, it will replace the helper text. Can be a node or a string.
 
+For best practices, use a `ref` on the input to enhance usability when handling an `errorMessage`.
```

Comment:
Maybe we can move this sentence to go right after `While React Magma provides the error styling, it is up to the consumer app to handle the validation` and we can elaborate a little more

It would read something like:

> While React Magma provides the error styling, it is up to the consumer app to handle the validation. We recommend using a `ref` on the input for accessibility. 
> 
> For short forms with an error, clicking submit should bring the focus back to the input with an error. For long forms, we recommend using an alert to combine the errors and focus should be moved to the alert. See example in Form.